app/controllers/application_controller.rb

```diff
@@ -296,7 +296,7 @@ def saved_redirect_url_or(default)
 
   def generic_exception(exception)
     logger.warn "Operation FAILED: #{exception}"
-    logger.debug exception.backtrace.join("\n")
+    logger.error exception.backtrace.join("\n")
```

Comment:
Updated: I kept 2 commits there: one for the silencers, the second for the logging of backtraces